packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -325,9 +374,9 @@ const Dropdown = ({
             />
           </Button>
         </Selector>
-
         {isOpen && (
           <OptionsList selected={selectedItem !== null} {...getMenuProps()}>
+            <Option onClick={() => selectItem(null)} />
```

Comment:
I wonder if this is the best API, since there's some cases that we need to omit the possibility of clearing the selection. As well as in some cases the user might need to see some value to the clear option, such as the default behavior for `<select>`
![image](https://user-images.githubusercontent.com/28108272/140419511-b955760b-9342-4e1a-b900-9ce0a3dce73c.png)


I believe we could recreate this Dropdown API (which will generate a breaking change) to achieve something as in [MUI](https://mui.com/components/selects/), for instance.

I'd like to hear everyone thoughts on this, so cc/ @ggdaltoso @tcK1. We can also bring this to further discussion including more people